### ISSUE TITLE
Terraform Crash: empty structure "access_logs."

### ISSUE THREAD

**Issue by jvasallo** [hashicorp/terraform#11572]
Hi there,

### Terraform Version
v0.8.5

### Affected Resource(s)
Not sure currently, We made no changes, just upgraded to latest TF. It appears to be related to walking aws_route53_record and trying to compare for access_logs? It could also be related to aws_elasticache_cluster as that is a dependency.

### Terraform Configuration Files
```hcl
#######
# Route53 CNAME for ElasticCache
#######
resource "aws_route53_record" "memcached_endpoint" {
  zone_id = "${var.private_dns_zone}"
  name = "memcached.application.date.env.company.com"
  type = "CNAME"
  ttl = "5"
  records = ["${element("${split(":","${aws_elasticache_cluster.memcached.configuration_endpoint}")}",0)}"]
}

#########################
#            ElastiCache               #
#########################
resource "aws_elasticache_cluster" "memcached" {
    cluster_id = "memcached"
    engine = "memcached"
    engine_version = "1.4.24"
    node_type = "cache.r3.large"
    num_cache_nodes = 2
    parameter_group_name = "default.memcached1.4"
    port = 12345
    subnet_group_name = "${aws_elasticache_subnet_group.cache_subnet.name}"
    security_group_ids = ["${aws_security_group.app_b.id}", "${aws_security_group.office_seattle.id}"]
}
```

### Debug Output
See Crash.log (let me know if you need more info)

### Panic Output
https://gist.github.com/jvasallo/cac90ce45fb4e7e6639d01744d900273

### Expected Behavior
Terraform should apply successfully.

### Actual Behavior
Terraform appears to panic post upgrade to v0.8.5. 

### Steps to Reproduce
1. `terraform apply`

### Important Factoids
N/A

### References
N/A

**Comment by jvasallo** [comment]
To help narrow down the issue, downgrading to Terraform v0.8.4 seems to have fixed the panic. I don't have time currently to dig into the issue more, but if this is not resolved later, I might take a stab at it. Sorry :(

**Comment by jvasallo** [comment]
After troubleshooting this further, it was not memcached related at all. The issue was we have an ELB we created through TF, an engineer manually added access_logs in the AWS Console to the ELB, when we run TF it was panicing about the diffs. I removed it from the ELB manually and things started working as expected.

I expect Terraform to complain about this, but it should toss a panic. Is that correct? I will test this issue in an environment that isn't production soon :joy:
